chronosQ-reservation-impl-hbase/src/main/java/flipkart/cp/convert/chronosQ/impl/hbase/HbaseSchedulerStore.java

```diff
@@ -133,7 +141,7 @@ public Long remove(String value, long time, int partitionNo) throws SchedulerExc
                 resultScanner.close();
             releaseHTableInterface(hTable);
         }
-        return entries;
+        return entries.stream().map(e -> new SchedulerData(e, e)).collect(Collectors.toList());
```

Comment:
Same as before : `e` is the value, which makes this logic incorrect.  We should instead change `List<String> entries = new ArrayList<>();` to `List< SchedulerData>` and populate correctly